CMakeLists.txt

```diff
@@ -144,6 +144,8 @@ option(EXECUTORCH_BUILD_COREML "Build the Core ML backend" OFF)
 
 option(EXECUTORCH_BUILD_CUSTOM "Build the custom kernels" OFF)
 
+option(EXECUTORCH_BUILD_CUSTOM_OPS_AOT "Build the custom ops lib for AOT" OFF)
```

Comment:
This name is too similar to `EXECUTORCH_BUILD_CUSTOM`. If you ask a random person what the different is between `EXECUTORCH_BUILD_CUSTOM` and `EXECUTORCH_BUILD_CUSTOM_OPS_AOT` they probably wouldn't have a good answer.

Really, `EXECUTORCH_BUILD_CUSTOM` should probably have a more descriptive name. If that's too disruptive, please update the help strings for both options to make it more clear how they differ, and help users understand which one to enable and when.